protocol/v2/ssv/validator/non_committee_validator.go

```diff
@@ -250,6 +257,13 @@ func (ncv *CommitteeObserver) processMessage(
 			}
 		}
 	}
+
+	// Remove older slots container
+	thresholdSlot := currentSlot - 12 // won't get new messages after 12 slots
```

Comment:
Can you please comment on why 12?

Reply:
we allow attestations i think up to 32 slots late

Reply:
@moshe-blox yeah, actually it's 34 (32 + lateSlotAllowance). I just thought more about it and it's not clear for me why this would be 12. If we change it to 34, let's get it from network config as we are preparing for the local testnet. Also, we can export `lateSlotAllowance` from message validation (although we'll probably need to think about a more convenient way to use as this is not the only place where we'd need 34)

Reply:
We've decided to implement a naive solution for now where we just loop on the map (which should always be 34 len max) and remove any slot that is => current slot - 32 .
deployed it to watch how it behaves since it's done every message, though the map size should be pretty small so it might not have a huge impact.